notebooks/105-language-quantize-bert/105-language-quantize-bert.ipynb

```diff
@@ -7,8 +7,8 @@
     "id": "rQc-wXjqrEuR"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.            model_name=f'{ir_model_xml}.stem',
</pre><p>please remove model_name (it used in CLI for IR serialization name)</p><p>Also ir_model_bin is not used in code anymore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/941/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>